collections/_release_4_5/entry-platforms-apple-export-to-visionos.md

```diff
@@ -0,0 +1,22 @@
+---
+type: entry
+section: platforms
+subsection: apple
+rank: 1
+importance: 3
+anchor: export-to-visionos
+title: Export to visionOS
+text: |
+  You can now export for Apple's XR platform, making Godot projects compatible with the Apple Vision Pro.
+
+  This makes visionOS the first officially-supported platform integration in about a decade.
```

Comment:
I assume it's meant to say that it's the first added platform in a while? Right now it sounds a bit misleading, as if it's the only supported platform or something. (or maybe it's just me)

Reply:
It should mention it is not (yet) a full XR support, but only support for the planar window apps. Otherwise, it might be misleading.